cli/src/com/redhat/ceylon/common/tools/AbstractTestTool.java

```diff
@@ -92,9 +92,11 @@ public void setVersion(String version) {
         this.version = version;
     }
     
-    @Option(longName = "tap")
-    @Description("Enables the Test Anything Protocol v13.")
-    public void setTap(boolean tap) {
+    @OptionArgument(argumentName = "file")
```

Comment:
I thought we decided that was broken because of tests printing to stdout, and that's why we needed the file? I don't really see that defaulting to broken functionality is very helpful to the user. But I'm not really a user of this at the moment, so maybe I'm not qualified to comment.